ssh_certificate_parser/__init__.py

```diff
@@ -150,7 +150,7 @@ def from_bytes(cls, byte_array):
         unknown, blob = take_pascal_bytestring(blob)
         raw_ca, blob = take_pascal_bytestring(blob)
         ca_cert_type, raw_ca_rest = take_pascal_string(raw_ca)
-        if ca_cert_type == 'ssh-rsa' or ca_cert_type == 'ecdsa-sha2-nistp384':
+        if ca_cert_type in ( 'ssh-rsa', 'ecdsa-sha2-nistp256', 'ecdsa-sha2-nistp384'):
```

Comment:
nit: let's kill off this extra space here, better yet, formatting this like the tuple list above where each item is on its own line would be a good idea.

```suggestion
        if ca_cert_type in ('ssh-rsa', 'ecdsa-sha2-nistp256', 'ecdsa-sha2-nistp384'):
```